res/controllers/Hercules DJ Control MP3 e2.bulk.xml

```diff
@@ -1,13 +1,12 @@
 <?xml version='1.0' encoding='utf-8'?>
-<MixxxControllerPreset mixxxVersion="1.11+" schemaVersion="1">
+<MixxxControllerPreset mixxxVersion="2.0+" schemaVersion="1">
     <info>
-        <name>Hercules DJ Control MP3 e2 / MP3 LE / Glow</name>
-        <author>Neale Pickett</author>
-        <description>Hercules DJ Control MP3 e2 / MP3 LE / Glow mapping. These three controllers have identical controls and share the same mapping.</description>
+        <name>Hercules DJ Control MP3 e2</name>
```

Comment:
I had the impression that these are basically the same controller. Any reason you removed it?

Reply:
Yes you're right, I wasn't sure about this then I removed it. I'll add it again.